### DISCUSSION TITLE
Mixed Content

### DISCUSSION THREAD

**Discussion by kaaax0815** [envault/envault#55] [Q&A]
I use Envault behind a SSL Reverse Proxy. The problem with that is that I get mixed content errors because it is trying to load all external contents (css, favicon, etc) over http

```
Mixed Content: The page at 'https://[censored]/setup' was loaded over HTTPS, but requested an insecure stylesheet '[censored]/css/app.css?id=a9fe95b17d9c58af8394'. This request has been blocked; the content must be served over HTTPS.
```

I use Envault in Docker with CapRover with help of #53

**Comment by danharrin** [comment]
Does your `APP_URL` use `https`?

**Reply by kaaax0815** [reply]
This is my `APP_URL` https://envault.[censored]/
I also set `APP_ENV` to debug if that matters
